### ISSUE TITLE
Machine Learning

### ISSUE THREAD

**Issue by lukaJevtic1** [lukaJevtic1/marketing-sales-machine-learning#8]
### Train/test split

- Set seed
- 80/20 split
- Check target distribution in train/test
- Prevent data leakage

Preprocessing pipeline

- Imputation
- Dummy encoding
- Numeric normalization where necessary
- Ensure transformations fit only on training data
- Validate factor levels

### Cross-validation

- 5-fold CV
- Define evaluation metrics
- Use CV only on training set
- Record mean and SD

###  Baseline model

- Mean prediction
- Calculate MAE
- Calculate RMSE
- Calculate R²

###  Linear Regression

- Train model
- Evaluate
- Interpret coefficients
- Check assumptions

**Comment by Srujana047** [comment]
Hi! I'd like to work on this issue if it's still available. Could you please assign it to me?

**Comment by lukaJevtic1** [comment]
> Hi! I'd like to work on this issue if it's still available. Could you please assign it to me?

Hi, this project is closed type, it is the work of two people.  if you are open to any future cooperation, leave me a contact, all the best.

**Comment by Srujana047** [comment]
Thank you for letting me know. I appreciate the clarification.
